### ISSUE TITLE
Enhancement Shaman legendary stack inaccurate

### ISSUE THREAD

**Issue by flapjack831** [Bloodmallet/bloodytools#24]
https://bloodmallet.com/chart/shaman/enhancement/legendaries/castingpatchwerk

Claims Witch Doctor's Wolf bones is +13.3% DPS while [FW+UW+Asc] Doom Winds is +3.36% dps

Running sims of both profiles:
https://www.raidbots.com/simbot/report/9bpz9QnYB3BhfbmKCnVTKP
https://www.raidbots.com/simbot/report/oTGd2dijBuLCN4L257tR2y

Witch Doctor's Wolf Bones confirmed at +13.3% dps, but [FW+UW+Asc] Doom Winds is +11.2% dps over its base profile (or +4.72% dps over the base profile used for the other legendaries) - neither of which is consistent with the reported percentage gain on the bloodmallet stack.

**Comment by flapjack831** [comment]
Additionally, it looks like [Asc] Witch Doctor's Wolf Bones is reporting as -2.83% dps, which is an odd way to report it since the legendary isn't DPS-negative for any talent configuration. It's possible that [Asc] Witch Doctor's Wolf Bones is 2.83% less dps than the default profile, but that difference is due to talent selection, which is not the same thing as the contribution being provided by the legendary for that profile.

**Comment by Bloodmallet** [comment]
Greetings,
I checked out your links and recreated the simulations locally. I think I found multiple reasons why the values don't match what you expect. I'm not sure yet if that is even "fixable" on the website, yet. Here is the list of differences I found:

1. [FW+UW+Asc] Doom Winds uses the following talent string right now: `2111133` ([file](https://github.com/Bloodmallet/bloodytools/blob/dev/bloodytools/simulations/legendary_special_cases.yml#L9)), the simulation you posted uses `2211133` which indeed provides a higher result
  - this is indeed easily fixable and will empower the legendary in the chart by roughly 50% if you want to change that
 2. All legendaries and exceptions are compared to the same baseline profile like you wrote in your second post. The chart does try to answer how each legendary performs compared to the current base profile (without legendary). And for that reason I think the WitchDoctor's Wolf Bones special case might just be wrong to include there. Unless it's valuable to see that it performs worse than the base profile without legendary.

Following up on the second point, what would the value be in seeing the gain of a legendary for its most affected talent combination (I'ma ssuming `3111133` is specifically great for this legendary?)? How is that relevant, because I thought everyone is striving for most overall damage, or well, burst, but certainly not "I do less damage overall, but my legendary affects my damage the most in this setup"? Sorry for not understanding yet :(
